src/nm_window.h

```diff
@@ -168,6 +168,34 @@ enum nm_color {
     NM_COLOR_RED         = 4
 };
 
+/* possible lowercase keys (decimal)
```

Comment:
There is no need to add commented code blocks.
Moreover, checkpatch complains about this block

Reply:
I removed them.